tox.ini

```diff
@@ -21,12 +21,42 @@ deps = -rdev-requirements.txt
 commands = coverage run --parallel -m pytest {posargs}
 
 
+# Uses default basepython otherwise reporting doesn't work on Travis where
+# Python 3.6 is only available in 3.6 jobs.
+[testenv:coverage-report]
+deps = coverage
+skip_install = true
+commands =
+    coverage combine
+    coverage report
+
+
 [testenv:flake8]
 basepython = python3.6
 deps = flake8
 commands = flake8 src tests setup.py conftest.py docs/conf.py
 
 
+# Runs when invoking tox without arguments and formats the files.
```

Comment:
A bit unexpected, isn't it?
Also, doesn't `tox` run all of `envlist` then?

Reply:
1. Maybe?  But in my own practice, I found it very useful.
2. It does, but the now removed `isort-check` was not part of the `envlist` – hence the distinction.